docker-compose-teams.yaml

```diff
@@ -87,12 +87,30 @@ services:
       - match
       - guacamole
 
+  team-roboteamtwente:
+    image: yuhanuncitgez/robocup:latest
```

Comment:
Could you share the Dockerfile with me? I'm not sure if you build the image correctly, based on what docker hub shows me here: https://hub.docker.com/layers/yuhanuncitgez/robocup/latest/images/sha256-392bd9a0b9a7b4055abdbaeb107907de92e49b912023f190ce8bfd0bc12485b4?context=explore

Reply:
Actually I wasn't quite sure how exactly the image should be set up, I basically just got our AI running inside a container and pushed it. 

Merging this would just allow us to change the image at any time, though :)


that's the dockerfile: https://github.com/RoboTeamTwente/roboteam_suite/blob/robocup/2021/robocup/Dockerfile

Reply:
ah, I see. That was not how it was intended :D 
Have a look at this section: https://github.com/RoboCup-SSL/ssl-simulation-setup#preparations-for-teams
You Dockerfile should start with `FROM robocupssl/ubuntu-vnc:latest`.

You only need to install and build dependencies in this Dockerfile. Everything that requires root. The actual software, especially the part that will change over time, can/should be build inside the running container. Have you tried out on of the existing team containers locally? You'll see that you will jump into a small desktop environment.

Reply:
And yes, you can change the image later, but it will only be updated when I rebuild the containers on the servers, which will probably not happen during the competition, except if someone asks me for it.

Reply:
I have not tried that yet, I currently don't have access to a Linux machine but that does make more sense.

Thank you I'll update the docker image.

Reply:
Docker works on Windows and Mac as well ;) You would need to install Bash and Python on Windows for the setup scripts, but that should also be easy.

Reply:
Ah actually windows doesn't support subdomains on localhost, we came across this issue on discord. But yeah I'll try it :)

Reply:
ah, yes. But that should also be solvable by entering the sub domains manually to your hosts file: http://digitalpbk.blogspot.com/2007/01/making-subdomains-on-localhost.html

i.e.
```
127.0.0.1 field-a.localhost
127.0.0.1 vnc.field-a.localhost
127.0.0.1 gc.field-a.localhost
...
```

Reply:
Yuhanun mentioned that he updated docker image: https://hub.docker.com/layers/yuhanuncitgez/robocup/latest/images/sha256-42fbd279f2c9face23992cebaab1a3e70be43c894231758e3cc53756e4553505?context=explore
Personally dont understand it, but he asked if you could check if the image is correct now.